Project/RAFinder/app/src/main/java/edu/rosehulman/rafinder/model/Hall.java

```diff
@@ -7,9 +7,9 @@
     private String name;
     private Floor[] floors;
 
-    public Hall(String name, Floor[] floors){
-        this.name=name;
-        this.floors=floors;
+    public Hall(String name, Floor... floors) {
```

Comment:
changed to "..." syntax, as it's a bit easier to work with and still accepts/is an array.